tools/project_manager/manage_image_sources.php

```diff
@@ -100,7 +99,7 @@
         else
         {
             output_header('', NO_STATSBAR);
-                if ($new)
+                if ($count > 0)
```

Comment:
Is this right? You replace `!$new` above with `$count > 0` and `$new` here with the same `$count > 0`.

Reply:
Semantically it's the same. $count is the result of a `SELECT COUNT(*)` statement which is always non-negative. `$new` was set if the count was 0, so if `$count > 0` then we have found an already existing image with the same ID.

Reply:
No, Chris is right. You've changed the logic here such that the request for approval will happen for *existing* records (`$count > 0`) when it needs to happen for *new* records (`$count == 0`).

Reply:
Oh blast. I'm an idiot. Thanks both. Updated.